packages/workflow/src/features/registration/handler.ts

```diff
@@ -198,32 +197,21 @@ export async function createRegistrationHandler(
     const resBundle = await sendBundleToHearth(payload)
     populateCompositionWithID(payload, resBundle)
 
-    if (
-      [
-        Events.REGISTRAR_BIRTH_REGISTRATION_WAITING_EXTERNAL_RESOURCE_VALIDATION,
-        Events.REGISTRAR_DEATH_REGISTRATION_WAITING_EXTERNAL_RESOURCE_VALIDATION,
-        Events.REGISTRAR_MARRIAGE_REGISTRATION_WAITING_EXTERNAL_RESOURCE_VALIDATION
-      ].includes(event)
-    ) {
-      // validate registration with resource service and set resulting registration number now that bundle exists in Hearth
-      // validate registration with resource service and set resulting registration number
-      invokeRegistrationValidation(payload, request.headers)
-    }
     if (isEventNonNotifiable(event)) {
-      return resBundle
+      return { resBundle, payloadForInvokingValidation: payload }
```

Comment:
While I was doing something else, I noticed I started getting `FHIR did not send a valid response` when declaring a death event. Traced it to these changes here. 

I think this is what you meant, right?

```
{ ...resBundle, payloadForInvokingValidation: payload }
```